packages/atlas/src/views/playground/Playgrounds/PlaygroundReactionsComments.tsx

```diff
@@ -0,0 +1,246 @@
+import React, { useState } from 'react'
+
+import { TabItem, Tabs } from '@/components/Tabs'
+import { Button } from '@/components/_buttons/Button'
+import { Select, SelectItem } from '@/components/_inputs/Select'
+import { TextArea } from '@/components/_inputs/TextArea'
+import { TextField } from '@/components/_inputs/TextField'
+import { VideoReaction } from '@/joystream-lib'
+import { useJoystream } from '@/providers/joystream'
+import { useTransaction } from '@/providers/transactionManager'
+import { useUser } from '@/providers/user'
+import { ConsoleLogger } from '@/utils/logs'
+
+const TABS: TabItem[] = [
+  { name: 'React to video' },
+  { name: 'React to comment' },
+  { name: 'Create comment' },
+  { name: 'Edit comment' },
+  { name: 'Delete comment' },
+]
+
+export const PlaygroundReactionsComments: React.FC = () => {
+  const [selectedTabIdx, setSelectedTabIdx] = useState(0)
+  const [videoId, setVideoId] = useState('')
+
+  const getTabContents = () => {
+    const props: CommonProps = {
+      videoId,
+    }
+    switch (selectedTabIdx) {
+      case 0:
+        return <ReactToVideo {...props} />
+      case 1:
+        return <ReactToComment {...props} />
+      case 2:
+        return <CreateComment {...props} />
+      case 3:
+        return <EditComment {...props} />
+      case 4:
+        return <DeleteComment {...props} />
+    }
+  }
+
+  return (
+    <div>
+      <TextField value={videoId} onChange={(e) => setVideoId(e.target.value)} label="Video ID" />
+      <Tabs tabs={TABS} onSelectTab={setSelectedTabIdx} />
+      {getTabContents()}
+    </div>
+  )
+}
+
+type CommonProps = {
+  videoId: string
+}
+
+const ReactToVideo: React.FC<CommonProps> = ({ videoId }) => {
```

Comment:
There is no option to remove the reaction. Is it possible? I thought when someone clicks a second time on the Like button, we should remove the reaction, right?

Reply:
Yes, and that's also how you do in the QN, you need to send your like again and it will cancel the original one